source-browser.html

```diff
@@ -132,7 +145,8 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           $('#topics').append(
             $('<div class="topic flex-auto">').append(
               $('<div>').append(
-                $('<h2 class="inline-block m0">').text(name),
+                $('<h2 class="inline-block m0 relative heading">', {id: `${name}-space`}),
```

Comment:
Why not just `id: name` like before? The `space` suffix seems odd.

Reply:
I need to use a different id for the empty space above the h2 header, that's why I'm using name-space.